src/test/com/yetanalytics/flint/error_test.cljc

```diff
@@ -168,6 +176,51 @@
                 (map-indexed (fn [idx err] (err/scope-error-msg err idx)))
                 first)))))
 
+(deftest aggregate-error-msg-test
+  (testing "aggregates error messages"
+    (is (= "1 SELECT clause has both wildcard and GROUP BY!"
+           (->> '{:select   :*
+                  :where    [[?x ?y ?z]]
+                  :group-by [?x]}
+                (s/conform qs/query-spec)
+                v/collect-nodes
+                va/validate-agg-selects
+                err/aggregate-error-msg)))
+    (is (= "1 SELECT clause has both wildcard and GROUP BY!"
+           (->> '{:select   :*
+                  :where    [[?x ?y ?z]]
+                  :group-by [?x]}
+                (s/conform qs/query-spec)
+                v/collect-nodes
+                va/validate-agg-selects
+                err/aggregate-error-msg)))
```

Comment:
duplicate `is`

Reply:
Oh yeah the second one is supposed to be a sub-query thanks for pointing that out.